File: tests/eval_c/095.c
#include <string.h>
#include <assert.h>

void f095(const char* s) {
    int i = 0;
    while (i < strlen(s)) {
        assert(s[i] == 'a');
        i++;
    }
    assert(i == 0 || i == 1);
}

File: tests/oopsla25/095.py
#!/usr/bin/env python3

import sys

def f095(s: str):
  i = 0
  while i < len(s):
    assert s[i] == "a"
    i += 1
  assert i == 0 or i == 1

if __name__ == '__main__':
  my_string = ""
  if len(sys.argv) == 1:
    my_string = sys.stdin.read()
  else:
    fd = open(sys.argv[1], 'r')
    my_string = fd.read()
    fd.close()
  
  f095(my_string)
